x-pack/legacy/plugins/spaces/public/views/management/edit_space/customize_space/customize_space_avatar.tsx

```diff
@@ -29,9 +42,68 @@ class CustomizeSpaceAvatarUI extends Component<Props, State> {
     super(props);
     this.state = {
       initialsHasFocus: false,
+      imageUrl: props.space.imageUrl,
```

Comment:
If I'm understanding this correctly, `this.state.imageUrl` is always kept in sync with `this.props.space.imageUrl`. If so, can we remove `this.state.imageUrl` and just rely on `this.props.space.imageUrl` instead?

Reply:
Ok done.